bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -485,10 +485,10 @@ spec:
   provider:
     name: IBM
   relatedImages:
-    - image: icr.io/cpopen/common-service-operator:3.20.0
+    - image: icr.io/cpopen/common-service-operator:3.20.1
       name: COMMON_SERVICE_OPERATOR_IMAGE
     - image: icr.io/cpopen/cpfs/ibm-cs-webhook:1.16.0
       name: IBM_CS_WEBHOOK_IMAGE
     - image: icr.io/cpopen/cpfs/ibm-secretshare-operator:1.17.0
       name: IBM_SECRETSHARE_OPERATOR_IMAGE
```

Comment:
Could we add a replace here for 3.20? For example,
<img width="604" alt="Screen Shot 2022-08-12 at 1 33 52 PM" src="https://user-images.githubusercontent.com/29827416/184412842-97d4ebeb-5721-4a15-8a6f-436d45d7e255.png">
1

Reply:
nice catch! will do now